content/aws_marketplace.md

```diff
@@ -16,557 +16,120 @@ product = ["client", "workstation"]
-    authorization to access the setup page.
-
-3.  Fill out the setup form and submit it.
+Use this instance with Chef Workstation installed on your laptop or a separate AWS instance.
 
-4.  Follow the link and log into Chef Automate.
-
-## Configure the workstation
+{{< workstation >}}
 
-1.  Download and extract the `starter_kit.zip` file to a directory on
-    the workstation. Open a command prompt and change into the
-    `chef-repo` directory extracted from the starter kit. For example:
-
-    ```bash
-    cd ~/Downloads
-    unzip starter_kit.zip
-    cd starter_kit/chef-repo
-    ```
+If you need to run Automate with a high availability (HA) topology, please contact [Chef Support](https://www.chef.io/support).
 
-2.  {{% install_aws_chef_server_knife_client_list %}}
+## Installation
 
-## Configure backups
+Select the Chef Automate AWS Marketplace [CloudFormation template](https://aws.amazon.com/marketplace/pp/prodview-r26bs6uknftps?ref_=srh_res_product_title).
 
-Follow the Workflow
-[instructions]({{< relref "automate/backup/" >}}) for configuring
-backups.
+Every CloudFormation Stack deployment creates a new [Virtual Private Cloud](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html) (VPC).
 
-## Troubleshooting
+{{< note >}}
+AWS provides 5 VPCs per region. If you require more VPCs, please contact [AWS Support](https://aws.amazon.com/contact-us/).
+{{< /note >}}
 
-### Required ports
+### Specify the Stack Details
 
-The following are recommended security group rules for Chef Automate
-from the AWS Marketplace:
+1. Enter the following values for your deployment.
 
-<table>
-<colgroup>
-<col style="width: 12%" />
-<col style="width: 87%" />
-</colgroup>
-<thead>
-<tr class="header">
-<th>Port</th>
-<th>Description</th>
-</tr>
-</thead>
-<tbody>
-<tr>
-<td>443</td>
-<td>HTTPS for Chef Automate</td>
-</tr>
-<tr>
-<td>8989</td>
-<td>Git access for the delivery-cli and workflow</td>
-</tr>
-<tr>
-<td>22</td>
-<td>SSH</td>
-</tr>
-</tbody>
-</table>
+     - Stack Name: `Chef-Automate`
+     - EC2RootVolumeSize: `Default: 40`
+     - Instance Type:`Default: t2.xlarge`
+     - KeyName: _Enter your existing keypair_
+     - SecurityGroupCidrIp: `0.0.0.0/0`
+     - SubnetCIDR: `10.0.0.0/24`
+     - VpcCIDR: `10.0.0.0/16`
 
-### Change the hostname
+1. Select **Next** and create your Chef Automate deployment. This process can take several minutes.
 
-To update the hostname, do the following:
+1. Give Chef Automate an additional five minutes for all the services to start running.
 
-1.  Run `sudo -i` to gain administrator privileges.
+![Select next to create stack](/images/StackDetails.png "Stack Details")
 
-2.  Run `chef-marketplace-ctl hostname` to view the current hostname.
+## Post-Installation
 
-3.  Configure the `api_fqdn` in `/etc/chef-marketplace/marketplace.rb`
+1. Navigate to the AWS deployment **output** tab and locate the Chef Automate URL, user name, and password. You will need these in the next steps.
```

Comment:
The list items in this section are formatted as ordered and unordered list items, it would be better to go with ordered list items for all of them.